### PR TITLE
add meta option to doc

### PR DESCRIPTION
**Pull request by tabcat**
This documents the meta option available at db creation. Includes input type expected, the default for the option, and a description of what it does/affects.

Looking for feedback on wording especially.

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -115,6 +115,8 @@ Returns a `Promise` that resolves to [a database instance](#store-api). `name` (
 
 - `overwrite` (boolean): Overwrite an existing database (Default: `false`)
 - `replicate` (boolean): Replicate the database with peers, requires IPFS PubSub. (Default: `true`)
+- `meta` (object): An object to be added to the database manifest (learn more about the [db manifest](https://github.com/orbitdb/orbit-db/blob/master/GUIDE.md#address)). Data in the manifest affects the address of the db and cannot be changed after creation. It's recommended to store smaller objects and must be serializable to json. Read the object at instance property `db.options.meta`. Default: `undefined`.
+
 ```javascript
```
Comment (aphelionz) [inline API.md:120]: I know this isn't your job... In this PR but would you mind adding all of the above options to the JS example for posterity?

File: API.md [hunk 2]
```diff
@@ -115,6 +115,8 @@ Returns a `Promise` that resolves to [a database instance](#store-api). `name` (
 
 - `overwrite` (boolean): Overwrite an existing database (Default: `false`)
 - `replicate` (boolean): Replicate the database with peers, requires IPFS PubSub. (Default: `true`)
+- `meta` (object): An object to be added to the database manifest (learn more about the [db manifest](https://github.com/orbitdb/orbit-db/blob/master/GUIDE.md#address)). Data in the manifest affects the address of the db and cannot be changed after creation. It's recommended to store smaller objects and must be serializable to json. Read the object at instance property `db.options.meta`. Default: `undefined`.
```
Comment (aphelionz) [inline API.md:118]: How about:

- `meta` (object): An optional object [database manifest](https://github.com/orbitdb/orbit-db/blob/master/GUIDE.md#address)). Immutably stores any JSON-serializable key/value pair and outputs via `db.options.meta`. Default: `undefined`.

Comment (aphelionz) [reply API.md:118]: There's also the idea that these things will affect the DB address but I'm not sure if here specifically is the place to put that info

**Comment by aphelionz** [comment]
ha, whoops. Why close?

**Comment by tabcat** [comment]
I was worried that something else was included with this but it wasnt.

File: API.md (hunk 1 above)
Comment (tabcat) [reply API.md:120]: sure, should this be in a separate pr?

Comment (aphelionz) [reply API.md:120]: I wouldn't mind if it was in the same PR since it's doc-only

File: API.md (hunk 2 above)
Comment (tabcat) [reply API.md:118]: much more concise which is nice, i would slightly rather `key/value pair` -> `value` and maybe `outputs` -> `readable`?

Comment (aphelionz) [reply API.md:118]: yeah that's fine

**Comment by tabcat** [comment]
any last minute changes i should make?

**Comment by aphelionz** [comment]
I'm gonna merge this despite the CI errors since you clearly didn't introduce them here

**Pull request merged**